docs/archive/CHANGELOG.md

```diff
@@ -81,6 +82,8 @@
 * [artifact-manager] IAC-788 / Improved Handling of Empty vRA Blueprint Versions for vRA 8.12.x.
 * [artifact-manager] IAC-801 / Fix vRA catalog items appearing twice on multi page results.
 
+## v2.35.2
```

Comment:
Could you check this one, we don't have a 2.35.2 release, probably this is some mixup because of merge conflicts. This change is part of the 2.36.0 release:
<img width="1273" alt="image" src="https://github.com/vmware/build-tools-for-vmware-aria/assets/22296648/bab5091a-53e2-4ac0-9cef-af973d22bf4c">

In the current version of the changelog it is properly placed:
<img width="899" alt="image" src="https://github.com/vmware/build-tools-for-vmware-aria/assets/22296648/72a4f77c-73cf-4f2b-8d53-98779f7b6e57">

so I guess just removing the 2.35.2 shoud be enough

Reply:
Code updated.